networks/dev/docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
     environment:
       - ID=dev
       - LOG=${LOG:-provenanced.log}
-      - PIO_HOME=/provenance/nodedev
```

Comment:
If you are changing this then you may want to consider also updating the PIO_HOME in networks/dev/blockchain-dev/entrypoint.sh. That runs a provenanced init with /provenance/nodedev as the home directory.

Reply:
The PIO_HOME in entrypoint.sh makes the home directory in `build/nodedev`.   This is identical to how the localnet works.